packages/framework/theming-react-native/src/BaselinePlatformDefaults.ts

```diff
@@ -4,12 +4,13 @@ import { getStockWebPalette, ITypography, ISpacing } from '@uifabricshared/themi
 function _defaultTypography(): ITypography {
   return {
     sizes: {
-      small: 11,
+      small: 10,
       medium: 12,
       large: 14,
-      largePlus: 18,
-      extraLarge: 24,
-      huge: 32
+      largePlus: 16,
+      extraLarge: 20,
+      huge: 28,
+      giant: 42

```

Comment:
Otherwise, would I have to define new sizes for the new type ramp? Like "fluentSmall", "fluentMedium", "fluentLarge", etc?

Reply:
I just remembered these aren't px but typographical points - I'll change them to the appropriate values (e.g. medium should be '9' instead of '12' to reflect 9 tp === 12 px) . I'm also going to make these sizes reflect the sizes in the classic type ramp since we haven't added the new type ramp yet. We can decide how to handle naming for the new type ramp values later?